t/model/161-category.t

```diff
@@ -64,7 +64,7 @@ MT::Association->link( $ukawa,  $designer,        $blog );
 MT::Association->link( $ichikawa, $edit_categories, $second_blog );
 
 # Run tests
-my ( $app, $out );
```

Comment:
This $app is also unused because of my $app in L75.

Reply:
https://github.com/movabletype/movabletype/pull/776/commits/a6aad59f55a450e2027041959a5f92f47b6c9dff